packages/fast-development-site-react/app/components/button/index.ts

```diff
@@ -0,0 +1,13 @@
+import manageJss from "@microsoft/fast-jss-manager-react";
+import { ButtonStyles, default as BaseButton } from "./button";
+
+/*
+ * The type returned by manageJss type is very complicated so we'll let the
+ * compiler infer the type instead of re-declaring just for the package export
+ */
+/* tslint:disable-next-line:typedef */
+const Button = manageJss(ButtonStyles)(BaseButton);
+type Button = typeof Button;
+
+export default Button;
+export * from "./button";
```

Comment:
Is this a necessary addition for this PR? This seems like a lot of additional overhead to maintain for components that only exist to test on the dev site.

Reply:
I tend to think of additional overhead as a negative _only_ when it can impact things negatively, i.e. performance or upkeep. This actually aligns the test app in the documentation site more to how we do things "in the real world" (base or MSFT) so I think it has good value.

I added another component to fully show a directory without documentation (a blank README.md in this case, though it would work the same without a README.md as well) as a good proof. As an outside user, I would find value in thorough examples that align with real usage.